readme.md

```diff
@@ -232,6 +233,7 @@ Made with Electron.
 - [electron-input-menu](https://github.com/parro-it/electron-input-menu) - Context menu for input elements.
 - [chrome-tabs](https://github.com/adamschwartz/chrome-tabs) - Chrome like tabs.
 - [titlebar](https://github.com/kapetan/titlebar) - Emulate the OS X window titlebar.
+- [express-web-components](https://github.com/scramjs/express-web-components) - Declaratively build Express.js applications with web components.
```

Comment:
express-web-components is a collection of web components that are used in the renderer to make apps, but they are server-side apps. They aren't UI, just functionality, but they are designed to work with a headless Electron instance (accessed with the scram-engine).